src/napari_imagej/_module_utils.py

```diff
@@ -297,9 +299,10 @@ def _napari_module_param_additions(
     """Returns a set of parameters useful for napari functionality."""
     # additional parameters are in the form "name": (type, default value)
     additional_params: Dict[str, Tuple[type, Any]] = {}
-    output_item = module_info.outputs().iterator().next()
-    if not type_mappings().type_displayable_in_napari(output_item.getType()):
-        additional_params["display_results_in_new_window"] = (bool, False)
+    # Allow non-layer outputs to be displayed in a new window
+    for output_item in module_info.outputs():
```

Comment:
The point is: If *any* outputs cannot be displayed in a layer, we want to give users the option to spawn a new window with all non-layer results.

Maybe make that clear with a docstring.

Reply:
Docstring added in e1ae9c2